lesson15/index.js

```diff
@@ -76,8 +77,9 @@ showModalBtn.addEventListener("click", () => {
   showModal();
 });
 
-fetchBtn.addEventListener("click", () => {
+form.addEventListener("submit", (e) => {
   const inputNum = document.getElementById("js-input-number");
   const inputName = document.getElementById("js-input-name");
+  e.preventDefault();
   init(inputNum.value,inputName.value); 
```

Comment:
IMO:
This arguments were hard a little too detailed for me to see.
If you want to do it quickly now, I thought this might be a one way to deal with it.

```
const inputNumValue = document.getElementById("js-input-number").value;
const inputNameValue = document.getElementById("js-input-name").value;

init(inputNumValue,inputNameValue); 
```

↑ If we're going to do this.
I think you should argument of init function have to change in a good mood that too.

Reply:
I certainly think so!
So I fixed it!

bb7cd32